README.md

```diff
@@ -16,9 +16,14 @@ yarn add gatsby-source-flamelink
 
 ## How to use
 
-// In your gatsby-config.js
+In order to use the plugin you need to configure it to point to the Firebase instance that is using Flamelink.
```

Comment:
Updated the docs here a bit while at it.

Reply:
thanks!